maven-model-builder/src/test/java/org/apache/maven/model/validation/DefaultModelValidatorTest.java

```diff
@@ -604,17 +609,21 @@ public void testBadImportScopeClassifier()
                         "'dependencyManagement.dependencies.dependency.classifier' for test:a:pom:cls must be empty" );
     }
 
-    public void testSystemPathRefersToProjectBasedir()
+    public void testSystemPathRefersToProjectBasedirAndDeprecatedScope()
```

Comment:
I think it's better having different test cases. Keeping the old test cases and create new ones for the changed behaviour. That makes it safe not to break previous behaviour,